src/components/atoms/Badge/Badge.tsx

```diff
@@ -23,17 +23,17 @@ interface IBadgeProps extends IStyledBadgeProps {
 }
 
 const backgroundColors: IBgColors = {
-  confirmed: alertColors.veggy500,
-  default: alertColors.alerty900,
-  invalid: alertColors.danger500,
-  waiting: alertColors.alerty500,
+  confirmed: colors.semantic.success,
+  default: colors.neutral.dark,
+  invalid: colors.semantic.error,
+  waiting: colors.semantic.alert,
```

Comment:
I know it's not a concern of this PR but I think it would be nice to standardise these semantic names and maybe always use `success`, `default`, `error`, `alert` instead of `confirmed`, `invalid`, etc.

Reply:
I can open a separate PR and update them 👍